etc/testing/circle/run_release_tests.sh

```diff
@@ -48,4 +48,64 @@ pushd examples/opencv
 popd
 
 pachctl delete pipeline --all
-pachctl delete repo --all
\ No newline at end of file
+pachctl delete repo --all
+
+pushd examples/shuffle
+    pachctl create repo fruits
+    pachctl create repo pricing
+    pachctl create pipeline -f shuffle.json
+    pachctl put file fruits@master -f mango.jpeg
+    pachctl put file fruits@master -f apple.jpeg
+    pachctl put file pricing@master -f mango.json
+    pachctl put file pricing@master -f apple.json
+
+    # wait for everything to finish
+    pachctl wait commit "shuffle@master"
+
+    # check downloaded and uploaded bytes
+    downloaded_bytes=$(pachctl list job -p shuffle --raw | jq '.stats.download_bytes | values')
+    if [ "$downloaded_bytes" != "" ]; then
+        echo "Unexpected downloaded bytes in shuffle repo: $downloaded_bytes"
+        exit 1
+    fi
+
+    uploaded_bytes=$(pachctl list job -p shuffle --raw | jq '.stats.upload_bytes | values')
+    if [ "$uploaded_bytes" != "" ]; then
+        echo "Unexpected downloaded bytes in shuffle repo: $uploaded_bytes"
+        exit 1
+    fi
+
+    # check that the files were made
+    files=$(pachctl glob file "shuffle@master:**" --raw | jq '.file.path' -r)
+    expected_files=$(echo -e "/apple/\n/apple/cost.json\n/apple/img.jpeg\n/mango/\n/mango/cost.json\n/mango/img.jpeg")
+    if [ "$files" != "$expected_files" ]; then
+        echo "Unexpected output files in shuffle repo: $files"
+        exit 1
+    fi
+popd
+
+pachctl delete pipeline --all
```

Comment:
nit - since it's only 3 repos it's probably safer here to just delete those repos/pipelines instead of doing delete all on whatever cluster this happens to run on.